ui/app/templates/components/oidc/provider-list.hbs

```diff
@@ -0,0 +1,42 @@
+{{#each @model as |model|}}
```

Comment:
This is definitely a preference thing on my side, but you could name `model` -> `provider` and maybe make this template a bit easier read.

Reply:
This could certainly be confusing as the only change of its type, though.

Reply:
great idea!